src/ConsoleConnector/Commands/Options/ParameterValue.cs

```diff
@@ -1,18 +1,41 @@
-﻿namespace Autodesk.DataExchange.ConsoleApp.Commands.Options
+﻿using Autodesk.DataExchange.SchemaObjects.Components;
+
+namespace Autodesk.DataExchange.ConsoleApp.Commands.Options
 {
     /// <summary>
     /// Parameter Value command option.
     /// </summary>
     /// <seealso cref="CommandOption" />
     internal class ParameterValue : CommandOption
     {
+        public Autodesk.DataExchange.Core.Enums.ParameterDataType ParameterValueType { get; set; }
         public ParameterValue()
         {
             this.Description = "Specify value for parameter.";
         }
+
+        public override void SetValue(string value)
+        {
+            base.SetValue(value);
+            this.ParameterValueType = IdentifyDataType(value);
+        }
+
         public override string ToString()
         {
             return "ParameterValue[" + Description + "]";
         }
+
+        public static Autodesk.DataExchange.Core.Enums.ParameterDataType IdentifyDataType(string input)
```

Comment:
Is there any other way to figure out the dataType? userinput or for built in params, the schema itself?
I could enter value as 4 which can be parsed as int, but my expectation could be that its a double or string with value 4?

Reply:
Data type detection is removed and now we are taking data type as parameter argument.